bottlecap/src/proc/clock.rs

```diff
@@ -0,0 +1,20 @@
+use std::io;
+
+#[allow(clippy::cast_sign_loss)]
+#[cfg(not(target_os = "windows"))]
+pub fn get_clk_tck() -> Result<u64, io::Error> {
+    let clk_tck = unsafe { libc::sysconf(libc::_SC_CLK_TCK) };
```

Comment:
I'm kinda not sure if this code is going to cause problems, we are only checking if it's -1, there's no comments here for unexpected/other behavior. I'm blocking the PR so we can talk about this in person.